helm-cider-repl.el

```diff
@@ -74,35 +75,37 @@ If not a positive integer, don't truncate candidate, show all."
   "Candidates for Helm CIDER REPL history.
 
 Blank and duplicate candidates are excluded."
-  (cl-loop for c in (helm-fast-remove-dups cider-repl-input-history :test 'equal)
-           unless (string-blank-p c)
-           collect c))
+  (cl-loop
+     for c in (helm-fast-remove-dups cider-repl-input-history :test 'equal)
+     unless (string-blank-p c)
+     collect c))
 
 (defun helm-cider-repl--history-transformer (candidates _source)
   "Display only the `helm-cider-repl-history-max-lines'
 lines of candidate."
-  (cl-loop for c in candidates
-           for m = helm-cider-repl-history-max-lines
-           when (get-text-property 0 'read-only c)
-           do (set-text-properties 0 (length c) '(read-only nil) c)
-           for n = (with-temp-buffer (insert c) (count-lines (point-min) (point-max)))
-           if (and (integerp m)
-                   (> n m 0))
-           collect (cons (with-temp-buffer
-                           (insert c)
-                           (goto-char (point-min))
-                           (concat
-                            (buffer-substring
-                             (point-min)
-                             (save-excursion
-                               (forward-line helm-cider-repl-history-max-lines)
-                               (point)))
-                            "[...]"))
-                         c)
-           ;; Need to collect a cons b/c persistent action is executed on
-           ;; unpropertied string otherwise
-           ;; See: https://github.com/emacs-helm/helm/blob/v2.4.0/helm.el#L4999
-           else collect (cons c c)))
+  (cl-loop
```

Comment:
I think it's a bit uncommon to break `loop` in this manner, but I guess you did it to shorten the long lines.
Generally I dislike `loop` a lot and I think it usually produces somewhat hard to understand code (not to mention it's harder to debug macros).

Reply:
> I think it's a bit uncommon to break `loop` in this manner

I recently [started using `common-lisp-indent-function`](https://github.com/xiongtx/.emacs.d/commit/5c73d1f0c9fbe4fa23de5978d79d19a1bec80ae6) for Elisp to align forms for `cl-flet` etc. One consequence is that `cl-loop` forms no longer line up like:

```emacs-lisp
(cl-loop for x in foo
         for y in bar
         ...)
```

But like:

```emacs-lisp
(cl-loop for x in foo
   for y in bar
   ...)
```

Which is surprising b/c the former is [how CLTR formats `loop`](https://www.cs.cmu.edu/Groups/AI/html/cltl/clm/node244.html#SECTION003060000000000000000). Nonetheless, I find moving the first clause onto a new line and keeping things lined up more pleasing.

> Generally I dislike `loop` a lot

I've come to take a sort of sadistic pleasure in using `loop`, perhaps influenced by Helm (which uses more `loop` than all of Emacs...I recall @Wilfred counted at some point :nerd_face:).